tests/apps/binding_output/go.mod

```diff
@@ -2,6 +2,12 @@ module app
 
 go 1.15
 
-require github.com/gorilla/mux v1.7.3
+require (
+	github.com/dapr/dapr v0.11.3
```

Comment:
Why does dapr need to be a dependency of the app?

Reply:
It uses the dapr client to make the grpc calls:

```go
commonv1pb "github.com/dapr/dapr/pkg/proto/common/v1"
runtimev1pb "github.com/dapr/dapr/pkg/proto/runtime/v1"
```

```go
client := runtimev1pb.NewDaprClient(conn)

for _, message := range requestBody.Messages {
	body, err := json.Marshal(&message)

	log.Printf("Sending message: %s", body)
	req := runtimev1pb.InvokeBindingRequest{
		Name:      "test-topic-grpc",
		Data:      body,
		Operation: "create",
	}
	_, err = client.InvokeBinding(context.Background(), &req)
	if err != nil {
		log.Printf("Error sending request to GRPC output binding: %s", err)
		w.WriteHeader(http.StatusInternalServerError)
		w.Write([]byte("error: " + err.Error()))
		return
	}
}
```

You can see a similar pattern in the service_invocation app:
https://github.com/dapr/dapr/blob/master/tests/apps/service_invocation/app.go
https://github.com/dapr/dapr/blob/master/tests/apps/service_invocation/go.mod

Reply:
Can you use the go SDK instead?

Reply:
Makes sense. It did add the go-sdk to the top level dapr go.mod. If you like this, we can open another issue to change it in the service invocation tests/other places where it's used. I'm pretty sure most of the grpc tests do this.

Reply:
by adding go-sdk if there are changes in dapr not consistent with the version of go-sdk then wouldn't that cause the test to fail?

Reply:
@mukundansundar Yes, but that should not be a problem post v1. This is the same issue we have with new E2E tests with app in Java, Python and .Net. Just happens that the apps written in Go can conveniently compile with dapr runtime code.